randovania/games/dread/json_data/Dairon.txt

```diff
@@ -1586,7 +1621,9 @@ Extra - asset_id: collision_camera_018
   * Layers: default
   * Open Passage to EMMI Zone Exit North/Dock to Yellow EMMI Introduction
   > Pickup (Energy Part)
-      Missiles and Lay Bomb
+      All of the following:
```

Comment:
I would just remove the "Pickup (Energy Part)" -> "Door to EMMI Zone Exit North"  connection entirely - not sure why that was there in the first place. Just have the energy part require morph ball to get out via the weight tile.

Reply:
also this was the wrong area for it. this comment was placed on the opposite connection